src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -318,6 +319,14 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 		}
 	}
 
+	private getCurrentLinkAddress(): string {
+		if (document.getSelection().anchorNode.parentNode['protocol'] === 'file:') {
+			return document.getSelection().anchorNode.parentNode['pathname'] || '';
+		} else {
+			return document.getSelection().anchorNode.parentNode['href'] || '';
```

Comment:
>document.getSelection() [](start = 10, length = 23)

would this always return a valid anchorNode which will have a parentNode or shoukd be checking for nulls?

Reply:
If the link is selected as shown in the gif in the issue, then there should always be an anchorNode with a parentNode as far as I tested.